Packages/Ethereum/Sources/Ethereum/EthRpc1.swift

```diff
@@ -1921,14 +1921,15 @@ extension Eth.TokenAmount: Hashable {
 
 extension Eth.TokenAmount: CustomStringConvertible {
     public var description: String {
+        let decimalSeparator = Character(Locale.current.decimalSeparator ?? ".")
```

Comment:
does it breaks tests for the "Ethereum" Xcode scheme?

Reply:
no